HPXMLtoOpenStudio/measure.rb

```diff
@@ -1819,10 +1825,35 @@ def self.apply_shading(model, window_or_skylight, shading_polygon, parent_surfac
       end
       shading_surface.setTransmittanceSchedule(shading_schedules[trans_values].schedule)
 
-      # Adjustment to default view factor is used to reduce ground diffuse solar
-      avg_trans_value = trans_values.sum(0.0) / 12.0 # FUTURE: Create EnergyPlus actuator to adjust this
-      default_vf_to_ground = ((1.0 - Math::cos(parent_surface.tilt)) / 2.0).round(2)
-      sub_surface.setViewFactortoGround(default_vf_to_ground * avg_trans_value)
+      # EMS to actuate view factor to ground
+      actuator = OpenStudio::Model::EnergyManagementSystemActuator.new(sub_surface, *EPlus::EMSActuatorSurfaceViewFactorToGround)
+      actuator.setName("#{sub_surface.name.to_s.gsub(' ', '_').gsub('-', '_')}_actuator")
```

Comment:
This is fragile since it is using the subsurface name directly but only handling a subset of characters that would break EMS. I was able to generate en E+ error by changing an HPXML window to "Window+North", which resulted in:
`Set Window+North_actuator = 0.5*fixedwindow_shading_coefficient, !- Program Line 1`
and
```
   ** Severe  ** EnergyManagementSystem:Actuator="WINDOW+NORTH_ACTUATOR", Invalid variable name entered.
   **   ~~~   ** ...Name; Names used as EMS variables cannot contain "+" characters.
```
Instead I suggest just using an index: window1_actuator, window2_actuator, etc. You'll have to update the tests too.